microsoft-edge/webview2/how-to/publish-uwp-app-store.md

```diff
@@ -0,0 +1,139 @@
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 04/18/2022
+---
+# Publish a UWP WebView2 app to the Microsoft Store
+
+A _UWP WebView2 app_ is a Universal Windows Platform (UWP) app that hosts the WebView2 control.  To make a UWP WebView2 app widely available, you can publish it to the Microsoft Store, by following the steps below.
+
+For general background information about publishing to the Windows Store, see [Publish Windows apps and games](/windows/uwp/publish/), at Windows App Development documentation > UWP.  The steps below provide the specific details about how to publish a UWP WebView2 app.
+
+
+<!-- ====================================================================== -->
+## Step 1: Configure a Microsoft Store account in Visual Studio
+
+*  Make sure you have a Microsoft Store account configured in Microsoft Visual Studio.  See [Get registered](https://developer.microsoft.com/microsoft-store/register/), about developer accounts for the Microsoft Store.
+
+Then continue with the steps below.
+
+
+<!-- ====================================================================== -->
+## Step 2: Test the non-packaged app
+
+<!-- Why test these particular aspects?  What other aspects?  What's the focus here? -->
+
+*  Make sure the app runs correctly on startup, and works correctly during shutdown sleep.
+
+
+<!-- ====================================================================== -->
+## Step 3: Package the app
+
+<!-- Why do we need to package the app? -->
+
+Package your app:
+
+1. Update the `package.appxmanifest` with all needed content, such as Visual Assets, Packaging name, and Application information.  If you don't do this, the Microsoft Store validation step will fail.  For reference, see [Validate your app package](/windows/msix/package/packaging-uwp-apps#validate-your-app-package), in _Package a desktop or UWP app in Visual Studio_.  Then continue with the steps below.
+
+1. At [Partner Center](https://partner.microsoft.com), select **Project** > **Publish** and then reserve an application name.
+
+1. Associate the application with the reserved name, by selecting **Project** > **Publish** > **Associate App with the Store**.
+
+1. Create the app package, by selecting **Project** > **Publish** > **Create App Packages**.  Select the target configurations.  Only 64-bit versions of the WebView2 Runtime will work in UWP:
+
+   ![Package Selection](publish-uwp-app-store-images/package-selection.png)
+
+1. Once the app package has been created for your configurations, you'll be presented with a dialog of information and have the option to run Windows App Certification Kit (WACK):
+
+   ![Windows App Certification Kit](publish-uwp-app-store-images/win-app-cert-kit.png)
+
+
+<!-- ====================================================================== -->
+## Step 4: Test the package
+
+<!-- Why do we need to test the package/ packaged app? -->
```

Comment:
@DavidShoe - Why do we need to test the package/ packaged app?  To make sure it still works, that dependencies are available when in the context of a different machine & directories?

Aiming for a good Sentence 1 for each h2 section.  @nishitha-burman